public/components/common/search/search.tsx

```diff
@@ -475,19 +474,21 @@ export const Search = (props: any) => {
                 />
               )}
             </EuiFlexItem>
-            <EuiFlexItem grow={false}>
-              <EuiToolTip position="bottom" content={needsUpdate ? 'Click to apply' : false}>
-                <EuiButton
-                  color={needsUpdate ? 'success' : 'primary'}
-                  iconType={needsUpdate ? 'kqlFunction' : 'play'}
-                  fill={!showQueryArea || fillRun} // keep fill on all the time if not using query assistant
-                  onClick={runChanges}
-                  data-test-subj="superDatePickerApplyTimeButton" // mimic actual timepicker button
-                >
-                  {needsUpdate ? 'Update' : 'Run'}
-                </EuiButton>
-              </EuiToolTip>
-            </EuiFlexItem>
+            {!showQueryArea && (
```

Comment:
Why are we removing the run button for query assistant? I don't see any other way the run is enabled, is this intentional?

Reply:
Nvm found it, the whole input.tsx file was hidden originally